File: proxads/src/main/java/com/proxglobal/rate/RatingDialogListener.java
package com.proxglobal.rate;

public abstract class RatingDialogListener {
    public void onSubmitButtonClicked(int rate, String comment) {};
    public void onLaterButtonClicked() {};
    public void onChangeStar(int rate) {};
    public void onRated() {};
    public void onDone() {};
}

File: proxads/src/main/java/com/proxglobal/rate/RatingDialogListener.kt
package com.proxglobal.rate

abstract class RatingDialogListener {
    open fun onSubmitButtonClicked(rate: Int, comment: String) {}
    open fun onLaterButtonClicked() {}
    open fun onChangeStar(rate: Int) {}
    open fun onRated() {}
    open fun onDone() {}
}